index.js

```
@@ -58,6 +60,46 @@ app.post("/github_push_webhook/:id", async (req, res) => {
   }
 });
 
+async function post_discord_url_to_db() {
+  try {
+    const data_2 = await pool.query(
+      `insert into github_discord_url (discord_url) values ($1)`,
+      [db_url]
+    );
+  } catch (err) {
+    console.log(err.message);
+  }
+}
+
+async function get_id_by_discord_url() {
+  try {
```

try block swallowing errors again